src/transport/raw/PeerAddress.h

```diff
@@ -214,6 +216,29 @@ class PeerAddress
     }
 
 private:
+    static PeerAddress FromString(char * addrStr, uint16_t port, Type type)
+    {
+        char * addrPart  = nullptr;
+        char * scopePart = nullptr;
+        Inet::IPAddress addr;
+
+        addrPart = strtok(addrStr, "%");
```

Comment:
Please consider using `strtok_r` for a generic utility like this that is not guaranteed to be invoked on only one thread.

Reply:
Done